### PR TITLE
Add code review 

### PR DESCRIPTION
**Pull request by ManarOmari**
we have to add at least 2 comments

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       href="https://fonts.googleapis.com/css?family=Inter"
       rel="stylesheet"
     />
-    <title>Portfolio: setup and mobile first</title>
+    <title>Setup and mobile first</title>
   </head>
   <body>
```
Comment (ManarOmari) [inline index.html:14]: Good job so far 👍 
✔️ No lintral errors 
✔️  Professional README file

**Review by ManarOmari — COMMENTED** [review]
there are some issues you have to update 

- [ ]  Add buttons events , like on click , on hover

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @ManarOmari 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Required Changes:
- [ ] Kindly add at least 2 inline comments, right now you have only 1.
- [ ] Inline comments should be about specific changes in the code while the summary of the review should be in the summary not in the inline comment (you did the opposite). Kindly fix this mistake 👍 
- [ ] It's better if the comments are polite suggestions instead of orders. For example:
  - This is a polite suggestion: "I think it would be better to use semantic tags"
  - This reads as an order: "Use semantic tags"
- [ ] Kindly provide an explanation, a "why", for each requested change.

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ManarOmari — COMMENTED** [review]
there are some issues you have to update 
I think you should follow Figma design and 
- [ ]  Add buttons events, like on click, on hover

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi again 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

## Required Changes:
- [ ] Please add at least 2 inline comments as I asked previously.
- [ ] Kindly make suggestions not orders.

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ManarOmari — COMMENTED** [review]
Good job so far 🏁 
✅ No lintral errors 
✅ Professional README file 
here are some changes that need to be implemented in order to proceed to the next project 👍

- [ ] Please follow all figma design .

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
       href="https://fonts.googleapis.com/css?family=Inter"
       rel="stylesheet"
     />
-    <title>Portfolio: setup and mobile first</title>
+    <title>Setup and mobile first</title>
   </head>
   <body>
     <header>
```
Comment (ManarOmari) [inline index.html:15]: [image](https://user-images.githubusercontent.com/34026019/186278659-cb3f6303-5c18-42ed-bea6-16bfbe395ff2.png)

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED 🟢 🟢 

Hi @ManarOmari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] You used the correct Github flow ✔️ 
- [ ] Good summary of your review ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**